pkg/gadgets/advise/seccomp/tracer/tracer.go

```diff
@@ -71,6 +73,10 @@ func (t *Tracer) install() error {
 	}
 
 	if err := spec.LoadAndAssign(&t.objs, nil); err != nil {
+		var ve *ebpf.VerifierError
+		if errors.As(err, &ve) {
+			log.Debugf("Verifier error: %+v\n", ve)
```

Comment:
It should use the logger from the gadget context, otherwise it won't be shown to the client terminal when using kubectl-gadget.